### PR TITLE
bug 1558228 linking to firewalld config

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -460,11 +460,18 @@ properly when the network interface for {product-title} has `NM_CONTROLLED=no`.
...
+other through specific ports that act as communication endpoints for certain
+processes or services.
+
+[NOTE]
+====
+While iptables is the default firewall, firewalld is recommended for new
+installations. You can enable firewalld by setting `os_firewall_use_firewalld=true` in xref:../../install_config/install/advanced_install.adoc#advanced-install-configuring-firewalls[the Ansible inventory file].
+====
 
 [NOTE]
```
Comment (vikram-redhat) [inline install_config/install/prerequisites.adoc]: @kalexand-rh - something went wrong here, I am assuming, while manually cherry picking. The note seems to be repeated.

**Comment by vikram-redhat** [comment]
@kalexand-rh - while it is possible to CP from branches other than master, to keep this a standard and to make sure we can keep track, let's restrict the use of the /cherrypick robot from only the master branch. I will update the manual to make this explicit.

File: install_config/install/prerequisites.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install_config/install/prerequisites.adoc]: The other branches are correct, and I'll open a new PR to fix the `enterprise-3.9` branch.

**Comment by kalexand-rh** [comment]
Fixing double note here: https://github.com/openshift/openshift-docs/pull/8417

**Pull request merged**